File: Python Module/Other/CreditsGUI.py
import Tkinter
from Other.project_utils import centered_window


def credits(game, name):
    global root
    global game_actions
    global username
    game_actions = game
    root = Tkinter.Tk()
    username = name
    root.title("Credits")
    root.geometry("1152x648")
    root.resizable(False, False)
    centered_window(root)

    root.config(bg="#36454F")

    label = Tkinter.Label(
        text="\nSaint Louis University\nSchool of Accountancy, Management, Computing, and Information Sciences\n" +
             "\nIT222L: Integrative Technologies\nFinal Laboratory Group Project\n\n" +
             "Group 4\nAromin, Eddyson Tristan\n" +
             "Arzadon, Maxwell John, A. \n" +
             "Del Rosario, Carliah Beatriz, M. \n" +
             "Lalwet, Carl Joshua, A. \n" +
             "Masaba, Jessalyn Mae, G.\n" +
             "Paguyo, Jan Ivan Ezra, D. \n" +
             "", font=("Arial Bold", 15), fg="white", bg="#36454F")
    label.pack(pady=20)

    return_to_menu_button = Tkinter.Button(root, text="Return to Main Menu", font=("Arial Bold", 15),
                                           command=go_to_main)
    return_to_menu_button.pack(pady=20)
    return_to_menu_button.config(background="white")
    return_to_menu_button.config(foreground="black")


def go_to_main():
    import MainMenu
    root.destroy()
    MainMenu.main_menu(game_actions, username)

File: Java Module/src/Client_Java/OtherUtilities/CreditsGUI.java
package Client_Java.OtherUtilities;

import Client_Java.GUITools.MoveListener;
import Client_Java.GUITools.StyledButton;

import javax.swing.*;
import java.awt.*;

public class CreditsGUI extends JFrame {

    public CreditsGUI(){
        JPanel panel = new JPanel();
        panel.setBackground(new Color(0x282828));
        panel.setPreferredSize(new Dimension(840, 521));

        JTextArea text = new JTextArea();
        text.setText("\nSaint Louis University\nSchool of Accountancy, Management, Computing, and Information Sciences\n" +
                "\nIT222L: Integrative Technologies\nFinal Laboratory Group Project\n\n"+
                "Group 4\nAromin, Eddyson Tristan\n" +
                "Arzadon, Maxwell John, A. \n" +
                "Del Rosario, Carliah Beatriz, M. \n" +
                "Lalwet, Carl Joshua, A. \n" +
                "Masaba, Jessalyn Mae, G.\n" +
                "Paguyo, Jan Ivan Ezra, D. \n" +
                "");
        text.setFont(new Font("Calibri", Font.BOLD, 20));
        text.setPreferredSize(new Dimension(800, 450));
        text.setColumns(20);
        text.setForeground(new Color(0xFFFFFF));
        text.setOpaque(false);
        panel.add(text);

        JButton back = new JButton("Return to Main Menu");
        back.setUI(new StyledButton());
        back.setForeground(Color.BLACK);
        back.setBackground(new Color(0xCBC9C9));
        back.addActionListener(e -> this.dispose());
        panel.add(back);

        MoveListener listener = new MoveListener(this);
        panel.addMouseListener(listener);
        panel.addMouseMotionListener(listener);

        this.setUndecorated(true);
        this.getRootPane().setWindowDecorationStyle(JRootPane.NONE);
        this.add(panel);
        this.setTitle("How To Play");
        setResizable(false);
        this.pack();
        this.setVisible(true);
        this.setLocationRelativeTo(null);
    }
}
